host/src/containers/GameSessionContainer.tsx

```diff
@@ -157,23 +157,23 @@ const GameSessionContainer = () => {
         setGameSession(response);
         checkGameTimer(response);
 
-  
+
       });
   };
 
   const checkGameTimer = (gameSession) => {
-      if (gameSession.currentState !== GameSessionState.CHOOSE_CORRECT_ANSWER && gameSession.currentState !== GameSessionState.CHOOSE_TRICKIEST_ANSWER){
-          setGameTimer(false);
-          setGameTimerZero(false);
-      }
-      else {
-          setGameTimer(true);
-          setGameTimerZero(false);
-      }
+    if (gameSession.currentState !== GameSessionState.CHOOSE_CORRECT_ANSWER && gameSession.currentState !== GameSessionState.CHOOSE_TRICKIEST_ANSWER) {
+      setGameTimer(false);
+      setGameTimerZero(false);
+    }
+    else {
+      setGameTimer(true);
+      setGameTimerZero(false);
```

Comment:
I know it's an existing line, but looks like this line is shared between two `if` and `else`, can be outside of the condition